SI/Sem.04/Pract.04/Task13.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+using namespace std;
+bool isPrime(int num)
+{
+	int squreRoot = sqrt(num);
+	if (num <= 1)
+	{
+		return false;
+	}
+	bool isPrime = true;
+	for (int i = 2; i <= squreRoot; i++) 
+	{
+		if (num % i == 0)
+		{
+			isPrime = false;
+			break;
+		}
+	}
+	return isPrime;
+}
+void allPrimeNums(int num)
+{
+	for (int i = 1; i < num; i++)
+	{
+		if (isPrime(i))
+		{
+			cout<< i <<" ";
+		}
+	}
+}
+
+bool isSumOfTwoPrimeNums(int num)
+{
+	bool isSum = false;
+	for (int i = 2; i < num; i++)
+	{
```

Comment:
Can be optimized => on line 36 if(!isPrime(i)) continue. Also you can skip all the even numbers except for two.